### PR TITLE
Use provenance domain from data in place page source footer

### PR DESCRIPTION
**Pull request by shifucun**
screenshot: https://staging.datacommons.org/dev/screenshot/8defc14

### CODE REVIEW

File: static/js/place/place_template.tsx [hunk 1]
```diff
@@ -749,7 +751,15 @@ class Chart extends Component<ChartPropType, ChartStateType> {
           <div id={this.props.id} className="svg-container"></div>
           <footer className="row explore-more-container">
             <div>
-              Data from <a href={config.url}>{config.source}</a>
+              <span>Data from </span>
+              {this.state.sources.map((source, index) => {
+                return (
+                  <span key={source}>
+                    <a href={"http://" + source}>{source}</a>
```
Comment (beets) [inline static/js/place/place_template.tsx:758]: is this safe to do for all the sources?

File: static/js/place/place_template.tsx [hunk 2]
```diff
@@ -749,7 +751,15 @@ class Chart extends Component<ChartPropType, ChartStateType> {
           <div id={this.props.id} className="svg-container"></div>
           <footer className="row explore-more-container">
             <div>
-              Data from <a href={config.url}>{config.source}</a>
+              <span>Data from </span>
+              {this.state.sources.map((source, index) => {
```
Comment (beets) [inline static/js/place/place_template.tsx:755]: we might need to add a filter to ensure uniqueness

**Review by beets — COMMENTED** [review]
This is a bit of a regression from before, particularly for EuroStat which now shows europa.eu, the home page for the EU. I know it'll be a bit of a backend change to do something better, but in the meantime, I wonder if we should add a map in python somewhere to remap some of these sources, e.g. eurostat, arcos. Can be a follow up too.

File: static/js/place/place_template.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/place/place_template.tsx:758]: checked this from bigquery Source.domain and the provenanceDomain should be domain only without http or https

File: static/js/place/place_template.tsx (hunk 2 above)
Comment (shifucun) [reply static/js/place/place_template.tsx:755]: Checked the source in StatsData is a set so here should be good: https://github.com/datacommonsorg/website/blob/master/static/js/shared/data_fetcher.ts#L59

**Comment by shifucun** [comment]
Yeah, agree we should have finer grade provenance url. Filed https://github.com/datacommonsorg/website/issues/429

File: static/js/place/place_template.tsx (hunk 1 above)
Comment (beets) [reply static/js/place/place_template.tsx:758]: could you add a todo, that we expect to get the full url?

**Review by beets — DISMISSED** [review]

**Comment by shifucun** [comment]
Changed to https and added todo, PTAL

**Review by beets — APPROVED** [review]
thanks for investigation, and making the change

**Comment by shifucun** [comment]
thx for review!

**Pull request merged**